var/spack/repos/builtin/packages/viennarna/package.py

```diff
@@ -42,4 +43,8 @@ def configure_args(self):
         if 'python@3:' in self.spec:
             args.append('--with-python3')
 
+        d = find('src', 'config.guess', recursive=True)
+        target = os.path.dirname(d[0])
+        copy(join_path('.', 'config.guess'), join_path(target))
```

Comment:
It looks like you are replacing `config.guess` in `src` with the `config.guess` in `.`. If I got that right, you can replace 

```
target = os.path.dirname(d[0])
copy(join_path('.', 'config.guess'), join_path(target))
```

with

```
copy(join_path('.', 'config.guess'), d[0])
```

* The filename you are looking for with `find` is `config.guess`, so you shouldn't have to take the `dirname` (unless the `copy` function provided by Spack is blocking you but I don't think it should)
* `join_path` shouldn't be required for the second argument to `copy` (since there is only one value)

Reply:
I replaced it with a simple command